jenkins-builds/7.0.UP00-19.03-mc

```diff
@@ -1,2 +1,26 @@
  abcpy-0.5.1-CrayGNU-19.03-python3.eb               --set-default-module
- CMake-3.14.1.eb 
```

Comment:
@kraushm Why did you remove `CMake`?

Reply:
It just moved a bit further down in the list.

Reply:
Sorry, I had overlooked that change